src/App.js

```diff
@@ -12,6 +13,7 @@ function App() {
         <Nav />
         <Routes>
           <Route path="/api/reviews" element={<Reviews />}></Route>
+          <Route path="/api/reviews/:review_id" element={<SingleReview />}></Route>
```

Comment:
I wouldn't include /api in your routes it would look neater as just /reviews/:review_id